angular-portfolio/src/app/name/name.component.scss

```diff
@@ -0,0 +1,38 @@
+
+.name {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    margin-top: -5%;
```

Comment:
I'd very much recommend against both absolute positioning and negative margins here, for the reason that they are very brittle and confusing rules, and may cause you issues in the future. Instead, it looks like you are centering this element. I'd recommend making a parent div for .name and giving it the rules:

display: flex;
align-items: center;
justify-content: center;

This will align it vertically and horizontally.